src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -315,6 +317,9 @@ def save_field_info(  # pylint: disable=too-many-arguments
     def field_has_info(self, key: str) -> bool:
         return self._storage.field_has_info(key)
 
+    def load_field_info(self, key: str) -> Any:
+        return self._storage.load_field_info(key)
```

Comment:
How come we don't know the return-type here?

Reply:
Guess we could always force a dict here